Base/Python/slicer/util.py

```diff
@@ -641,6 +641,43 @@ def array(pattern = "", index = 0):
 
 
 #
+# Qt
+#
+
+class QtWidgetMixin(object):
+  def __init__(self):
+    super(QtWidgetMixin, self).__init__()
+    self.Widget = None
+
+  def loadUI(self, path, fail_silently=False):
+    # Attempts to load and return a ui file that the given path points to.
+    # This method will raise an AssertionError if the ui file cannot be
+    # loaded and fail_silently is False (default).
+    import qt
+    qfile = qt.QFile(path)
+    qfile.open(qt.QFile.ReadOnly)
+    loader = qt.QUiLoader()
+    widget = loader.load(qfile)
+    if not fail_silently and not widget:
+      errorMessage = "Could not load UI file: " + str(path) + "\n\n"
+      raise AssertionError(errorMessage)
```

Comment:
In that case, I would recommend to pass the widget as a parameter to `get`:

```
def MyClass(QtWidgetMixin):
  def __init__(self):
    super(MyClass, self).__init__()
    firstPart = self.loadUI('/my_path/firstPart.ui')
    secondPart = self.loadUI('/my_path/secondPart.ui')

    self.Wigdet = qt.QWidget()
    layout = qt.QVBoxLayout()
    layout.addWidget(firstPartWidget)
    layout.addWidget(secondPartWidget)
    self.Widget.setLayout(layout)

    w1 = self.get(self.Widget, 'WidgetName_FromFirstPart')
    w2 = self.get(self.Widget, 'WidgetName_FromSecondPart')
```

By default, you could also automatically use `self.Widget`:

```
def MyClass(QtWidgetMixin):
  def __init__(self):
    super(MyClass, self).__init__()
    self.loadUI('/my_path/TopLevelWidget.ui')
    w = self.get(self.Widget, 'WidgetName_From_TopLevelWidget')
```